rules-tests/Php70/Rector/FuncCall/MultiDirnameRector/Fixture/fixture.php.inc

```diff
@@ -7,16 +7,6 @@ function multiDirname()
     dirname(dirname($path));
 
     new dirname(dirname(dirname($path)));
-
-
-// untouched
-    dirname(dirname($path, $level));
```

Comment:
Moved these cases to a new "skip" file with all the cases that should not be changed